source/tutorials/configure-selinux/configure-selinux.rst

```diff
@@ -0,0 +1,139 @@
+        ...
+
+    .. note::
+        1. The *permissive* mode only can be activated in **userdebug** or **eng** builds, as the user builds always operate in *enforcing* SELinux mode in Android.
+        2. After you change the *sepolicy* value in *mixins.spec*, you need to run the '*device/intel/mixins/mixin-update*' script to make the changes taking effect before starting the build.
+
+To change SELinux mode at runtime
+---------------------------------
+
+    Besides changing the SELinux operaion mode statically by editting the configuration file during the build time, users can also change the SELinux mode at runtime through *adb shell* commands.
+
+    * Get the current SELinux operation mode with ``getenforce`` command
+
+    .. code-block:: bash
+        :emphasize-lines: 2
+
+        $ adb shell
+        celadon:/ $ getenforce
+        Permissive
+
+
+    * Set the SELinux operation mode with ``setenforce`` command
+
+    .. code-block:: bash
+        :emphasize-lines: 3, 7
+
+        $ adb shell
+        celadon:/ $ su
+        celadon:/ # setenforce 0
+        celadon:/ # getenforce
+        Permissive
+
+        celadon:/ # setenforce 1
+        celadon:/ # getenforce
+        Enforcing
+
+    .. note::
+        1. You must run the ``setenforce`` command with *root* permission.
+        2. Changing the SELinux operation mode will return to its default mode specified in the *mixins.spec* file after system reboots.
+
+Add sepolicy rules for a module
+===============================
+
+When adding a new module to |C|, you may need to add relevant sepolicy rules, otherwise its operations may be blocked by SELinux. The following instructions take the '*rfkill*' module as an example, and show you the required steps to get accesses to the system resources.
+
+Add the initial sepolicy rules
+------------------------------
+
+#. Create a sub-folder '*rfkill*' under the *device/intel/project-celadon/sepolicy* directory for hosting the sepolicy rules files.
+
+#. Introduce the above folder to the sepolicy compiler by adding the following line to the board configuration overlay file '*device/intel/mixins/groups/rfkill/true/BoardConfig.mk*'.
+
+    .. code-block:: none
+
+        BOARD_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/rfkill
+
+#. Inside the sepolicy rules folder, create an initial file named '*file_contexts*' with the following content, to assign a file label for the *rfkill* executable file.
+
+    .. code-block:: none
+
+        /vendor/bin/rfkill-init.sh	u:object_r:rfkill_exec:s0
+
+#. Create a SELinux type enforcement file '*rfkill.te*' fo define the policy type and access control for the *rfkill* module:
+
+    .. code-block:: none
+        :emphasize-lines: 2, 4, 7
+
+        # Define a domain that rfkill process will run in
+        type rfkill, domain;
+        # Define a file type for the rfkill executable file, and assign this file type to it
```

Comment:
Suggested rewrite (if it doesn't change the meaning): 
Define a file type for the rfkill executable file and assign this file type to rfkill_exec, exec_type, file_type, and vendor_file_type;